.github/workflows/build-test.yml

```diff
@@ -1,23 +1,8 @@
 name: build-test
-on: [push]
+on: 
+  push:
 
 jobs:
   build-test:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v4
-      - name: Setup Go
-        uses: actions/setup-go@v4
```

Comment:
I do not see setup-go in the central actions. I also don't see python for the other stuff. Just curious here if that is a problem or it all works as expected without it?

Reply:
It still worked without any issues when I was testing. I'll take a deeper look at it.